spec/dummy/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationController < ActionController::Base
```

Comment:
@teoljungberg thanks for that link, though. I really like the idea of `bundler/inline`, and `gemfile { # ... }`.

I'll try those in the future when testing against multiple versions of Rails